doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -171,6 +171,7 @@ Serial LTE modem
   * ``#XMQTTCFG`` AT command to configure MQTT client before connecting to the broker.
   * The :ref:`CONFIG_SLM_AUTO_CONNECT <CONFIG_SLM_AUTO_CONNECT>` Kconfig option to support automatic LTE connection at start-up or reset.
   * The :ref:`CONFIG_SLM_CUSTOMER_VERSION <CONFIG_SLM_CUSTOMER_VERSION>` Kconfig option for customers to define their own version string after customization.
+  * A new optional parameter to the ``#XCARRIEREVT`` AT notification.
```

Comment:
Suggestion to add what paramerer and to which notification.

Reply:
Makes sense for the parameter to be added here, but I think it's better to let the users read the documentation for more details.